docker-compose-edinburgh-1.0.0.yml

```diff
@@ -100,7 +100,7 @@ services:
       - consul
 
   vault-worker:
-    image: edgexfoundry/docker-edgex-vault-worker-go:1.0.0
+    image: edgexfoundry/docker-edgex-vault-worker-go:1.0.0-dev
```

Comment:
Probably don't want the `-dev` tag in here. That's for local docker-compose work.

Reply:
will remove the -dev tag.